src/Round.js

```diff
@@ -0,0 +1,76 @@
+class Round {
+  constructor(generatedPlayers, clues) {
+    this.players = generatedPlayers;
```

Comment:
You don't need to be storing your players on every class -- you should store them once, in one place, and pass them through as arguments to methods wherever they're needed.